build_projects/dotnet-cli-build/PackageTargets.cs

```diff
@@ -230,6 +230,7 @@ public static BuildTargetResult GenerateNugetPackages(BuildTargetContext c)
                 dotnet.Pack(
                     projectFile,
                     "--no-build",
+                    "--serviceable",
```

Comment:
Yes, it does.

var serviceable = app.Option("-s|--serviceable", "Set the serviceable flag in the package", CommandOptionType.NoValue);